src/components/DateRangePickerInput.jsx

```diff
@@ -233,11 +241,14 @@ function DateRangePickerInput({
           {...css(styles.DateRangePickerInput_arrow)}
           aria-hidden="true"
           role="presentation"
+          onKeyDownTab={onEndDateTab}
         >
           {arrowIcon}
         </div>
       }
 
+      {datePickerIsAfterEndInput || children}
```

Comment:
I think this can be `{isStartDateFocused && children}`

Reply:
@majapw I'll try it! That's something I didn't notice until I added the other one, so makes sense as long as the input element is taken into account w/ it as well!